src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -1949,6 +1955,7 @@ bool isRefEscape
             GetInvocationArgumentsForEscape(
                 symbol,
                 receiver: null, // receiver handled explicitly below
+                receiverCloned: false,
```

Comment:
>false [](http://example.com/codeflow?start=32&length=5)

Why unconditionally ```false```?

Reply:
Because `receiver` is passed as `null`, so value of `receiverCloned` is not used.